ptychodus/__main__.py

```diff
@@ -27,17 +27,16 @@ def main() -> int:
 
     parser = argparse.ArgumentParser(
         prog=ptychodus.__name__.lower(),
-        description=f'{ptychodus.__name__} is a ptychographic reconstruction user interface')
-    parser.add_argument('-b',
-                        '--batch',
-                        action='store_true',
-                        help='run reconstruction non-interactively')
+        description=f'{ptychodus.__name__} is a ptychography analysis front-end')
+    parser.add_argument('-b', '--batch', action='store_true', \
+            help='run reconstruction non-interactively')
     parser.add_argument('-d', '--dev', action='store_true', help='run in developer mode')
-    parser.add_argument('-s',
-                        '--settings',
-                        action='store',
-                        type=argparse.FileType('r'),
-                        help='use settings from file')
+    parser.add_argument('-f', '--file-prefix', action='store', dest='prefix', \
```

Comment:
Does the file prefix replacement feature require the use of "$PREFIX" in the setting file? Otherwise, how does it determine what is prefix and what is not?